.taskcluster.yml

```diff
@@ -73,11 +73,13 @@ tasks:
         routes:
           - 'notify.irc-channel.#android-ci.on-any'
         scopes:
-          - queue:create-task:mobile-3/b-linux
-          - queue:scheduler-id:mobile-level-3
-          - secrets:get:project/focus/firebase
-          - secrets:get:project/focus/nimbledroid
-          - queue:route:notify.irc-channel.#android-ci.on-any
+          $let:
+            short_head_branch:
+              $if: 'event.ref[:11] == "refs/heads/"'
+              then: {$eval: 'event.ref[11:]'}
+              else: ${event.ref}
+          in:
+            - 'assume:repo:focus-android:branch:${short_head_branch}'
```

Comment:
Hm, I'm not sure if this is needed. I think we only need `branch:master`.

Reply:
This operates on any `master-push`, and the command checks out `event.after`, so I think that it's more general than just `master`

Reply:
Sounds good 👍